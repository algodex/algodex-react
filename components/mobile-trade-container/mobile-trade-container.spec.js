import React from 'react'
import renderer from 'react-test-renderer'
import MobileTradeContainer from '.'

it('MobileTradeContainer: default', () => {
  const component = renderer.create(< MobileTradeContainer />)
  const tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
