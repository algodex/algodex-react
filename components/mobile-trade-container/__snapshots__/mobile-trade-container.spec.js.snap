// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MobileTradeContainer: default 1`] = `null`;
